backend/codingchallenge/api/views/applications.py

```diff
@@ -227,7 +227,7 @@ def delete(self, request, *args, **kwargs):
         try:
             application = Application.objects.get(applicationId=self.kwargs["applicationId"])
 
-        except(KeyError, TypeError):
+        except(KeyError, TypeError, Application.DoesNotExist):
```

Comment:
Correct Exception would be `ObjectDoesNotExist`, but will be implemented by #239

Reply:
why shouldnt be Application.DoesNotExist correct? Every model (e.g. Application) inherits from the ObjectDoesNotExist Exception.

Reply:
just for equality reasons. We can also change every Exception to Application.DoesNotExist instead of ObjectDoesNotExist. But we should keep it equal.